docs/src/pages/lab/speed-dial/speed-dial.md

```diff
@@ -12,3 +12,7 @@ You can provide an alternate icon for the closed and open states using the `icon
 of the `SpeedDialIcon` component.
 
 {{"demo": "pages/lab/speed-dial/OpenIconSpeedDial.js"}}
+
+Actions tooltip can be set to always visible to prevent long press on mobile platform.
```

Comment:
"The `SpeedDialAction`s tooltips can be be displayed persistently so that users don't have to long-press in order to see the tooltip on touch devices.

It is enabled here across all devices for demo purposes, but in production could use the `isTouch` logic to conditionally set the property."